lldb/source/Plugins/ObjectFile/Minidump/MinidumpFileBuilder.cpp

```diff
@@ -1024,6 +1027,8 @@ Status MinidumpFileBuilder::AddMemoryList_32(
     error = AddData(data_up->GetBytes(), bytes_read);
     if (error.Fail())
       return error;
+
+    progress.Increment();
```

Comment:
might be nice to add detail before the ReadMemory above in case the first range is huge and give a description:
```
std::string descriotion = ...; // Put string in that give us the memory range as a string
process.Increment(1, description);
```